app/services/bulk_upload/lettings/year2022/row_parser.rb

```diff
@@ -530,53 +528,36 @@ def duplicate_check_fields
     ].compact
   end
 
-  def validate_location_related
-    return if scheme.blank? || location.blank?
-
-    unless location.scheme == scheme
-      block_log_creation!
-      errors.add(:field_5, "Scheme code must relate to a location that is owned by owning organisation or managing organisation")
-    end
-  end
-
   def location
     return if scheme.nil?
 
     @location ||= scheme.locations.find_by_id_on_multiple_fields(field_5)
   end
 
-  def validate_location_exists
+  def validate_related_location_exists
     if scheme && field_5.present? && location.nil?
-      errors.add(:field_5, "Location could not be found with the provided scheme code", category: :setup)
+      block_log_creation!
+      errors.add(:field_5, "Scheme code must relate to a scheme that is owned by the owning organisation or managing organisation", category: :setup)
```

Comment:
Should this be talking about the location?

Reply:
I'm pretty sure this was an error in copy before - this is for the 2022 form that only refers to management groups and old-core schemes (i.e. locations) so the error message should match that I think.